src/esw/brushed_motors/Controller.h

```diff
@@ -96,6 +96,10 @@ sent. This is to prevent multiple virtual Controller objects from
 trying to contact the same physical Controller object.)
 */
 
+struct LiveState {
+    std::string jointName;
+    inline static std::mutex liveMutex;
```

Comment:
Why exactly is this inline static? I don't think we need to do this

Reply:
I'm not sure why this is happening, but if I take out inline static it no longer builds.

Reply:
I think if you put the LiveState struct into the Controller class where it says public, it will build without the static

Reply:
it might be possible that you might need to end up creating a custom constructor the LiveState struct afterwards then.